docs/spec2017-tutorial.md

```diff
@@ -0,0 +1,635 @@
+# Tutorial: Run SPEC CPU 2017 Benchmarks in Full System Mode with gem5art  
+
+## Introduction  
+In this tutorial, we will demonstrate how to utilize gem5art to run SPEC CPU 2017 benchmarks in gem5 full system mode. 
+The full example with all of the gem5art tutorials can be found [here](https://github.com/darchr/gem5art-experiments). 
+The scripts in this tutorial work with gem5art-* v0.3.1.  
```

Comment:
Are you sure this shouldn't be 0.4.1?

Reply:
I haven't updated the launch script. I'll do that later.